jsonschema-validation.xml

```diff
@@ -634,14 +606,24 @@
                             validation in the application, which is the recommended best practice.
                         </cref>
                     </t>
+                </section>
 
+                <section title="Format-Assertion Vocabulary">
+                    <t>
+                        When the Format-Assertion vocabulary is declared with a value of false,
```

Comment:
```suggestion
                        When the Format-Assertion vocabulary is declared with a value of true,
```
...

> implementations MUST provide full validation support for all of the formats
>                         defined by this specificaion. Implementations that cannot provide full
>                         validation support MUST refuse to process the schema.


I'm assuming this is just an oversight, but there are no requirements when the vocabulary is declared with a value of `false` (which is "support not required").

Reply:
✔️